src/k8-metadata-client/src/client.rs

```diff
@@ -65,8 +65,15 @@ pub struct ListArg {
 /// trait for metadata client
 pub trait MetadataClientError: Debug + Display {
     /// is not founded
+    #[deprecated(
+        since = "3.3.0",
+        note = "This method is no longer used. Use not_found instead"
+    )]
     fn not_founded(&self) -> bool;
```

Comment:
I think that here we can add the default behavior to return self.not_found(), so we dont have to implement that for ClientError and DoNothingError

Reply:
Thanks, I had a thought like this but forgot to look more into it somehow. Will fix

Reply:
update? Like to merge this.  We are doing release tomorrow

Reply:
I think that we can merge as it is now,  and then do that last cleanup after the release

Reply:
was busy at work, will take a look a bit later

Reply:
Should be fixed now